### DISCUSSION TITLE
No images parsed from iOS 15.6 iPhon 12

### DISCUSSION THREAD

**Discussion by mark7699** [abrignoni/iLEAPP#330] [General]
Using my own phone for testing and validation get WiFi Bluetooth etc data but no images or social media data . What am I doing wrong could there be issues on install? What would you need to see what I’m doing wrong is there a file I can send?

**Comment by abrignoni** [comment]
What type of extraction are you trying to process? Be aware most artifacts depend on having a full file system extraction available.

**Reply by mark7699** [reply]
iTunes backup using idevicbackup2

**Reply by abrignoni** [reply]
Most artifacts depend on having a full extraction available.

**Reply by mark7699** [reply]
What do you use to do full exstraction
